### ISSUE TITLE
ValueError: probabilities contain NaN

### ISSUE THREAD

**Issue by hellocatfood** [Analyzable-FR/patch-based-texture-inpainting#1]
I am attempting to inpaint on this image of flowers using the following mask

![input](https://user-images.githubusercontent.com/151947/222980462-add22c91-4c99-4d29-b4cb-d429e13b28be.jpg)

![mask](https://user-images.githubusercontent.com/151947/222980464-42eabfe7-6fc1-47e5-ba6e-66d12e150d2f.jpg)

To do this this python script was created by a user (not me).

[inpaint_test.zip](https://github.com/Analyzable-FR/patch-based-texture-inpainting/files/10892695/inpaint_test.zip)

This script fails with the following error

```
python inpaint_test.py 
/home/hellocatfood/.local/lib/python3.10/site-packages/patch_based_inpainting/inpaint.py:349: RuntimeWarning: invalid value encountered in divide
  probabilities = 1 - distances / np.max(distances)
Traceback (most recent call last):
  File "/home/hellocatfood/Desktop/inpaint_test.py", line 49, in <module>
	inpaint = pbts.resolve()
  File "/home/hellocatfood/.local/lib/python3.10/site-packages/patch_based_inpainting/inpaint.py", line 314, in resolve
	patch_id = np.random.choice(ind, 1, p=probabilities)
  File "mtrand.pyx", line 935, in numpy.random.mtrand.RandomState.choice
ValueError: probabilities contain NaN
```

This same user reported that the script probably fails because the mask has multiple areas for masking. They reported that adding the following code to inpaint.py on line 311, before `:` fixes it

`and (0.0 < np.sum(dist) < np.inf)`

**Comment by bgallois** [comment]
The script fail because no patches with a valid probability can be computed ie. no similar path were found, not because the mask contains several areas to inpaint. The `and (0.0 < np.sum(dist) < np.inf)` workaround will inpaint the area, selecting random patches from the list of all patches.
A most graceful way to handle this would be to set the probability to 0 instead of nan.

**Comment by bgallois** [comment]
Fixed with 3b0f73621f42de8c13ea2ec312c4de6a47025a74. Will be release on pip with 0.0.3.
